src/olympia/reviewers/tests/test_views.py

```diff
@@ -972,13 +972,21 @@ def _test_get_queue(self):
         for addon in self.expected_addons:
             self.get_queue(addon)
 
-    def _test_queue_count(self, position, name, count):
-        response = self.client.get(self.url)
+    def _test_queue_count(self, name, position, count, totalqueues,
+                          per_page=None):
+        args = {'per_page': per_page[0]} if per_page else {}
+        response = self.client.get(self.url, args)
         assert response.status_code == 200
         doc = pq(response.content)
-        a = doc('.tabnav li a').eq(position)
-        assert a.text() == '%s (%s)' % (name, count)
-        assert a.attr('href') == self.url
+        links = doc('.tabnav li a')
+        link = links.eq(position)
+
+        assert links.length == totalqueues
+        assert link.text() == '%s (%s)' % (name, count)
+        assert link.attr('href') == self.url
+        if per_page:
+            assert doc('.data-grid-top .num-results').text() == (
+                u'Results %s\u20131 of %s' % per_page)
```

Comment:
Is there a parameter missing? You have two substitutes.

Reply:
What kind of refactoring are you imagining? I guess I could do `(per_page[0], per_page[1])`, or 

```javascript
page_count, page_max = per_page
assert doc('.data-grid-top .num-results').text() == (
    u'Results %s\u20131 of %s' % (page_count, page_max)
```

Reply:
Yes, having two separate arguments might be easier for other developers using that function.

Reply:
Turns out I actually only need one of those arguments.